app/javascript/mastodon/features/ui/components/media_modal.js

```diff
@@ -40,6 +40,11 @@ export default class MediaModal extends ImmutablePureComponent {
     this.setState({ index: (this.props.media.size + this.getIndex() - 1) % this.props.media.size });
   }
 
+  handleChangeIndex = (e) => {
+    const index = Number(e.currentTarget.getAttribute('data-index'));
+    this.setState({ index: (index) % this.props.media.size });
```

Comment:
* I do not think it needs parentheses around `index`.
* Is there any case `% this.props.media.size` matters?

Reply:
I referred to the existing code.
https://github.com/tootsuite/mastodon/blob/master/app/javascript/mastodon/features/ui/components/media_modal.js#L32

I removed them.